Add the package script and the docker file.

movie-travel-code opened the pull request:
Given that 'go-langserver' uses a manually-updated mechanism for
tracking versions, we must change the hardcode URL to download the
packaged 'go-langserver'.

zfy0701 commented on script/package.sh:
```diff
@@ -0,0 +1,87 @@
+                      mkdir packages
+                      for PLATFORM in linux darwin windows
+                      do
+                        curl -OL https://github.com/elastic/go-langserver/releases/download/v0.1.3/go-langserver-\$PLATFORM-amd64.tar.gz
```
could you make version as a variable and put on top few lines of the file?

movie-travel-code replied:
Sure, that totally makes sense!

movie-travel-code replied:
ping @zfy0701

Commit:
Add the package script and the docker file.

Given that 'go-langserver' uses a manually-updated mechanism for
tracking versions, we must change the hardcode URL to download the
packaged 'go-langserver'.
